src/abbrs.rs

```diff
@@ -99,8 +99,14 @@ impl Abbreviation {
         if !self.matches_position(position) {
             return false;
         }
-        if !self.commands.is_empty() && !self.commands.contains(&command.to_owned()) {
```

Comment:
would have been nice to add a test case to the commit message,
or (even better) to tests/checks/abbr.fish,
so reviewers don't have to read through the issue.